website/guides/sgb_border.md

```diff
@@ -0,0 +1,178 @@
+# Adding a custom SGB border
+
+This document was written to aid developers of DMG compatibile homebrew with adding Super Game Boy borders.
+
+We wil see how:
+
+- Detect whether we are running on a SGB
+- Transfer the border's tiles
+- Transfer and display the border's tilemap and palettes
+
+<small>
+
+Written by [sylvie (zlago)](https://zlago.github.io/me/), idea (and minor help) by [valentina (coffee bat)](https://coffeebat.neocities.org/), reviews and improvements by [ISSOtm](https://eldred.fr) and [avivace](https://github.com/avivace).
+
+</small>
+
+## Enabling SGB features
+
+Before we can do anything else, we must first specify in the header that this game is an SGB game.
+Otherwise, the SGB BIOS will ignore any packets we send, and we won't even be able to detect when we are running form a SGB.
```

Comment:
```suggestion
Before we can do anything else, we must first specify in the header that this game is aware of SGB features.
Otherwise, the SGB BIOS will ignore any packets we send, and we won't even be able to detect when the program is running on SGB.
```